src/components/ContactPage/ContactForm/index.tsx

```diff
@@ -0,0 +1,134 @@
+                <br />
+                available by phone, email, and social media for your
+                convenience.
+              </p>
+            </div>
+            <div className="flex md:flex-row md:items-center md:space-x-4">
+              <div className="flex items-center space-x-2 mr-10">
+                <span>+1-123-456-7890</span>
+                <FaArrowRight className="text-blue-500" />
+              </div>
+              <div className="flex items-center space-x-2">
+                <span>info@damienbraunphotography.com</span>
+                <FaArrowRight className="text-blue-500" />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="mb-12 flex flex-row justify-between">
+          <div className="flex flex-col">
+            <h3 className="uppercase mb-2 text-3xl">Send Me A Message</h3>
+            <p className="text-lg mb-8">
+              Have a specific inquiry or message for us? Please use the <br />
+              contact form below, and we'll get back to you promptly.
+            </p>
+          </div>
+          <div className="justify-self-start">
+            <form className="space-y-6">
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+                <div>
+                  <label
+                    htmlFor="first-name"
+                    className="block text-sm font-medium text-gray-400"
+                  >
+                    First Name
+                  </label>
+                  <input
+                    type="text"
+                    id="first-name"
+                    name="first-name"
+                    className="mt-1 p-3 block w-full bg-gray-800 border-gray-700 rounded-md shadow-sm text-white"
+                    placeholder="First Name"
+                  />
+                </div>
+                <div>
+                  <label
+                    htmlFor="last-name"
+                    className="block text-sm font-medium text-gray-400"
+                  >
+                    Last Name
+                  </label>
+                  <input
+                    type="text"
+                    id="last-name"
+                    name="last-name"
+                    className="mt-1 p-3 block w-full bg-gray-800 border-gray-700 rounded-md shadow-sm text-white"
+                    placeholder="Last Name"
+                  />
+                </div>
+                <div>
+                  <label
+                    htmlFor="email"
+                    className="block text-sm font-medium text-gray-400"
+                  >
+                    Email
+                  </label>
+                  <input
+                    type="email"
+                    id="email"
+                    name="email"
+                    className="mt-1 block p-3 w-full bg-gray-800 border-gray-700 rounded-md shadow-sm text-white"
+                    placeholder="Email Address"
+                  />
```

Comment:
I'd move both label and input to common components as it seems that they're highly reusable